docs/Manifests.md

```diff
@@ -1,6 +1,190 @@
 # Manifests
 
-## AppManifest
+## AppManifest Specification
+
+The AppManifest YAML file defines your application's components, routes, resources, and lifecycle events.
+
+### Components top-level element
+
+An AppManifest must declare a `components` root element as a list of objects containing configuration for each component in your application. At least one component object must be defined.
+
+```yaml
+components:
+  - name: webserver
+    ...
+  - name: api
+    ...
+```
+
+<!-- <details> -->
+
+#### name
+
+Defines the name of the component. Each component `name` must be unique. **Required**.
+
+#### type
+
+Specifies the type of the component; must be one of `service`, `task`, or `static`. **Required**.
+
+#### image
+
+Sets the base image from which the component is built. **Required**.
+
+#### port
+
+The container port to expose.
+
+#### root
+
+Establishes the docker build context, setting the host working directory for `copy` and `run` build step commands.
+
+#### build
+
+Defines a series of `steps` to be executed in creating the component image. Steps are defined as an array of mappings, where the mapping key is the step instruction and mapping value are arguments for that instruction.
+
+Steps:
```

Comment:
Hmmm, this `Steps:` line sort of disappears when skimming the documentation file. Thinking it might be best if they wasn't simply plaintext.... Maybe test out making it bold?

(Note to myself) I'm also noticing there aren't significant visual distinction between h4, h5 and plaintext element when this section of document in the Console. I might need to make styling adjustments in the Console to improve the legibility here.

<img width="627" alt="Screenshot 2022-11-10 at 12 43 36 PM" src="https://user-images.githubusercontent.com/48101033/201169313-de8ab688-74ac-42bb-92f5-abdcc02d9bcb.png">